### PR TITLE
Fixes related to upload file

### PR DESCRIPTION
**Pull request by sladan-hedvig**
- Moved data from upload files to url
- Grid view update
- Open picker from details
- Seperate files to different query
- Change file upload to be modal

## Checklist

- [ ] Dark/light mode verification
- [ ] Landscape verification
- [ ] iPad verification
- [ ] iPod/iPhone 12 mini/iPhone SE verification

### CODE REVIEW

File: Projects/Claims/Sources/Helpers/FileUploadManager.swift [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+                        atPath: FileUploadManager.uploadFolderPath.appendingPathComponent($0).relativePath
+                    )
+                })
+            } else {
+                try FileManager.default.createDirectory(
+                    at: FileUploadManager.uploadFolderPath,
+                    withIntermediateDirectories: true
+                )
+            }
+        } catch let _ {
```
Comment (julia-andersson) [inline Projects/Claims/Sources/Helpers/FileUploadManager.swift:32]: Do we need this if the catch doesn't do anything?

File: Projects/Claims/Sources/Models/File.swift [hunk 2]
```diff
@@ -41,37 +42,76 @@ struct ImagePicker: UIViewControllerRepresentable {
...
                                     mimeType: .JPEG,
                                     name: "\(Date().currentTimeMillis).jpeg",
-                                    source: .data(data: data)
+                                    data: data,
+                                    thumbnailData: thumbnailData,
+                                    extension: "jpeg"
                                 )
-                            self.parent.filesSelected(file)
-
+                            //                            ImageCache.default.store(image, forKey: id)
```
Comment (julia-andersson) [inline Projects/Claims/Sources/Models/File.swift:79]: Remove

File: Projects/hCoreUI/Sources/Views/DocumentPreview.swift [hunk 3]
```diff
@@ -43,3 +47,34 @@ extension DocumentPreview {
...
+    //            decidePolicyFor navigationResponse: WKNavigationResponse,
+    //            decisionHandler: @escaping (WKNavigationResponsePolicy) -> Void) {
+    //
+    //        if let mimeType = navigationResponse.response.mimeType {
+    //            // do some thing with the MIME type
+    //        } else {
+    //            // response has no MIME type, do some special handling
+    //        }
+    //        decisionHandler(.allow)
+    //    }
```
Comment (julia-andersson) [inline Projects/hCoreUI/Sources/Views/DocumentPreview.swift:79]: Remove

**Review by julia-andersson — CHANGES_REQUESTED** [review]

File: Projects/Claims/Sources/Helpers/FileUploadManager.swift (hunk 1 above)
Comment (sladan-hedvig) [reply Projects/Claims/Sources/Helpers/FileUploadManager.swift:32]: Yeah, this should always success so it should be fine

**Review by julia-andersson — APPROVED** [review]

**Pull request merged**